opslevel/resource_opslevel_service.go

```diff
@@ -70,13 +69,6 @@ func resourceService() *schema.Resource {
 				ForceNew:    false,
 				Optional:    true,
 			},
-			"owner_alias": {
```

Comment:
`owner_alias` would need to also be removed from `opslevel/datasource_opslevel_service.go`

Reply:
I am refraining from removing `owner_alias` in the datasource, because doing so would cause the user to be unable to retrieve the owner's alias, since `owner` in the datasource maps to an ID.